platform/ceph/BUILD.bazel

```diff
@@ -0,0 +1,156 @@
+load("//ceph:unpack.bzl", "basename", "unpack_filegroup")
+
+BINARIES = [
+    "ceph-authtool",
+    "ceph-bluestore-tool",
+    "ceph-conf",
+    "ceph-dencoder",
+    "ceph-mgr",
+    "ceph-mon",
+    "ceph-objectstore-tool",
+    "ceph-osd",
+    "ceph-syn",
+    "rados",
+    "radosgw",
+    "radosgw-admin",
+    "radosgw-es",
+    "radosgw-object-expirer",
+    "radosgw-token",
+    "rbd",
+    "rbd-mirror",
+    "rbd-nbd",
+    "rbd-replay",
+    "rbd-replay-prep",
+]
+
+LIBRARIES = [
+    "libcephfs.so.2",
+    "librados.so.2",
+    "libradosstriper.so.1",
+    "librbd.so.1",
+    "librgw.so.2",
+    "librgw_admin_user.so.0",
+    "ceph/libceph-common.so.0",
+    "ceph/compressor/libceph_lz4.so.2",
+    "ceph/compressor/libceph_snappy.so.2",
+    "ceph/compressor/libceph_zlib.so.2",
+    "ceph/compressor/libceph_zstd.so.2",
+    "ceph/crypto/libceph_crypto_isal.so.1",
+    "ceph/crypto/libceph_crypto_openssl.so",
+    "ceph/erasure-code/libec_clay.so",
+    "ceph/erasure-code/libec_isa.so",
+    "ceph/erasure-code/libec_jerasure.so",
+    "ceph/erasure-code/libec_jerasure_generic.so",
+    "ceph/erasure-code/libec_jerasure_sse3.so",
+    "ceph/erasure-code/libec_jerasure_sse4.so",
+    "ceph/erasure-code/libec_lrc.so",
+    "ceph/erasure-code/libec_shec.so",
+    "ceph/erasure-code/libec_shec_generic.so",
+    "ceph/erasure-code/libec_shec_sse3.so",
+    "ceph/erasure-code/libec_shec_sse4.so",
+    "rados-classes/libcls_cas.so.1",
+    "rados-classes/libcls_hello.so.1",
+    "rados-classes/libcls_journal.so.1",
+    "rados-classes/libcls_kvs.so.1",
+    "rados-classes/libcls_lock.so.1",
+    "rados-classes/libcls_log.so.1",
+    "rados-classes/libcls_lua.so.1",
+    "rados-classes/libcls_numops.so.1",
+    "rados-classes/libcls_otp.so.1",
+    "rados-classes/libcls_rbd.so.1",
+    "rados-classes/libcls_refcount.so.1",
+    "rados-classes/libcls_rgw.so.1",
+    "rados-classes/libcls_sdk.so.1",
+    "rados-classes/libcls_timeindex.so.1",
+    "rados-classes/libcls_user.so.1",
+    "rados-classes/libcls_version.so.1",
+]
+
+cmake_external(
+    name = "ceph",
+    binaries = BINARIES,
+    cache_entries = {
+        "WITH_MANPAGE": "off",
+        "WITH_PYTHON3": "off",
+        "WITH_LTTNG": "off",
+        "WITH_EMBEDDED": "off",
+        "WITH_TESTS": "off",
+        "WITH_CEPHFS": "off",
+        # using system boost shaves off about 25% of the build time in my tests
+        "WITH_SYSTEM_BOOST": "on",
+        "WITH_RADOSGW_BEAST_FRONTEND": "on",
+        "WITH_FUSE": "off",
+        # disabled because the dashboard requires npm, which requires nodeenv,
+        # which requires working pip in a virtualenv, which is not working for
+        # some reason relating to DNS resolution.
+        # (and it's too much hell to make it functional for me to want to deal
+        # with this mess right now.)
+        "WITH_MGR_DASHBOARD_FRONTEND": "off",
+    },
+    lib_source = "@ceph//:source",
+    make_commands = [
+        "make -j12",
```

Comment:
why 12?

Reply:
Half the number of CPUs on RHOMBI, which allows for nice parallelism without completely stepping on everything bazel is doing.

But yeah, this should probably be scaled automatically based on # of CPUs. The real problem is that it's a really bad thing to be running two different job servers at once; but we don't have much of an option, because without parallelism, this step takes hours, and there's no current way to have make use bazel's build server.